### ISSUE TITLE
cant find mpv-1.dll

### ISSUE THREAD

**Issue by blurymind** [jaseg/python-mpv#66]
Here is the backtrace

```
  File "f:/WORK/PYTHON/webmTest/MPV/webm.py", line 2, in <module>
    import mpv
  File "F:\WORK\PYTHON\WinPython-64bit-3.6.3.0Zero\python-3.6.3.amd64\lib\site-packages\mpv.py", line 31, in <module>
    backend = CDLL('mpv-1.dll')
  File "F:\WORK\PYTHON\WinPython-64bit-3.6.3.0Zero\python-3.6.3.amd64\lib\ctypes\__init__.py", line 348, in __init__
    self._handle = _dlopen(self._name, mode)
```
I have it placed in the directory where the python script is (webm.py)

**Comment by docoprusta** [comment]
Hi,

I assume your error message is  "OSError: [WinError 193] %1 is not a valid Win32 application" (it is not on your posted backtrace but I tried to use it with portable python and I received the same error message as you with the oserror).

The problem is that you are trying to use a 32 bit dll with a 64 bit python.

You have two options:
1. Use [32 bit portable python](https://sourceforge.net/projects/winpython/files/WinPython_3.6/3.6.3.0/WinPython-32bit-3.6.3.0Zero.exe/download)
2. [Build a 64 bit mpv-1.dll](https://github.com/mpv-player/mpv/blob/master/DOCS/compile-windows.md)

If you are not familiar with building mpv and you have no reason to use 64bit python the first option is easier.

I hope this solves your problem. I am just a user of this project and I tried to use it with portable python as you and it throwed error with portable python 64 bit but it worked with portable python 32 bit for me.

I have not tried to build mpv for 64 bit but I assume it is also a good solution.
